tests/unit/deferred.py

```diff
@@ -193,9 +196,26 @@ def test_apply_networking(switch, nic1, nic2, network, fresh_database):
                                             channel='vlan/native',
                                             type='modify_port')
 
+    action_native3 = model.NetworkingAction(nic=nic3,
+                                            new_network=None,
+                                            channel='',
+                                            type='revert_port')
```

Comment:
see [this failure](https://travis-ci.org/CCI-MOC/hil/jobs/266135520#L2363) in my previous travis build where I made an incorrect assertion and that caused it to fail.